lib/fs/promises.js

```diff
@@ -402,7 +393,7 @@ async function lchown(path, uid, gid) {
   if (O_SYMLINK !== undefined) {
     const fd = await open(path,
                           O_WRONLY | O_SYMLINK);
-    return fchmod(fd, uid, gid).finally(fd.close.bind(fd));
+    return fchown(fd, uid, gid).finally(fd.close.bind(fd));
```

Comment:
Is it possible to add a test for this?

Reply:
/cc @nodejs/collaborators

Reply:
@mscdex In general, `fs/promises` has poor coverage and this is not the only method that lacks it completely. In the current state, I view that as a separate task. I mentioned that in the PR.

Ref: https://coverage.nodejs.org/coverage-b55a11d1b17b3e4b/root/fs/promises.js.html